install/build.py

```diff
@@ -112,10 +113,12 @@ def check_cudnn_version(compiler, settings):
 
     cudnn_version = int(out)
 
-    if cudnn_version < minimum_cudnn_version:
+    if not minimum_cudnn_version <= cudnn_version <= maximum_cudnn_version:
+        min_major = minimum_cudnn_version // 1000
+        max_major = maximum_cudnn_version // 1000
         utils.print_warning(
-            'cuDNN version is too old: %d' % cudnn_version,
-            'cuDNN v2 or newer is required')
+            'Unsupported cuDNN version: %d' % cudnn_version,
+            'cuDNN v%d<= and <=v%d is required' % (min_major, max_major))
```

Comment:
Actually a user cannot download old release if newer revision of cuDNN is released because old revision is removed from the download site. In such case, a user need to use lower minor version of cuDNN till Chainer supports the latest. It is inconvenient for users.
How about checking minor version? Usually revision release does not break APIs. In my opinion, this fix is not for guaranteeing supported version but for avoiding a user to use the newest  version. It always breaks compatibility.